templates/themes/app/parts/content_fields.html.twig

```diff
@@ -42,7 +42,22 @@
                         )
                     ) }}
                 {% else %}
-                    <a href="{{ path("ibexa.url.alias", { "contentId": field.value.reference }) ~ suffix }}"{% if css_class is not empty %} class="{{ css_class }}"{% endif %}{% if field.value.isTargetLinkInNewTab %} target="_blank"{% endif %}>{{ label }}</a>
+                    {{ render(
+                        controller(
+                            "ng_content::viewAction",
+                            {
+                                'contentId': field.value.reference,
+                                'viewType': 'link',
+                                'layout': false,
+                                'params': {
+                                    'label': label,
+                                    'suffix': suffix,
+                                    'css_class': css_class|default(''),
```

Comment:
`css_class` is already defaulted above, no need to repeat it here.

Reply:
fixed in cfc9c7d4e0a0383b681ca81747af4b56302b9d18